smpmgr/stat_management.py

```diff
@@ -0,0 +1,40 @@
+import asyncio
+from typing import cast
+
+import typer
+from rich import print
+from smpclient.requests.statistics_management import GroupData, ListOfGroups
+
+from smpmgr.common import Options, connect_with_spinner, get_smpclient, smp_request
+
+app = typer.Typer(name="stat", help="The SMP stat Management Group.")
```

Comment:
Please update the help to say "statistics" instead of "stat" to avoid ambiguity. "Stat" is Ok elsewhere since it matches the spec.